### DISCUSSION TITLE
pin code

### DISCUSSION THREAD

**Discussion by bodyboarder2528** [akopachov/flipper-zero_authenticator#112] [Q&A]
hi

apparently I set a pin yet I haven't so how do I reset it please?

**Comment by akopachov** [comment]
There is no way to reset\change pin once it is already set and you forgot it without loosing all the tokens. Otherwise such a pin won't make any sense.

However you can completely reset application including PIN, but keep in mind that you will loose all the tokens and settings you created

[How to change PIN](https://github.com/akopachov/flipper-zero_authenticator/wiki/How-to-change%5Crecover-PIN%3F)

[How to reset application](https://github.com/akopachov/flipper-zero_authenticator/wiki/How-to-reset-application%3F)

**Comment by bodyboarder2528** [comment]
i have added no tokens so its fine to reset
